### DISCUSSION TITLE
"missing" hooks "woocommerce_before_main_content" and "woocommerce_after_main_content"

### DISCUSSION THREAD

**Discussion by Axos11** [bootscore/bootscore#868] [General]
I just stumbled over the fact that the 2 woocommerce hooks woocommerce_before_main_content and woocommerce_after_main_content are not present in the woocommerce.php file and their hooked content was replaced manually. So i wanted to ask if there is any reason for that. 
(I added them back in and removed the boilerplate in my template I am working on right now and couldn't find any issues)

**Comment by crftwrk** [comment]
> So i wanted to ask if there is any reason for that.

There is no reason for this, it was simply overlooked. Would be great if you could quickly create a PR with the two hooks. We could merge it directly and release next Monday with planned 6.0.2.

Thank you for pointing this out.
